evutil_time.c

```diff
@@ -134,7 +134,10 @@ evutil_usleep_(const struct timeval *tv)
 	sleep(tv->tv_sec);
 	usleep(tv->tv_usec);
 #else
-	select(0, NULL, NULL, NULL, tv);
+	{
+		struct timeval tv2 = *tv;
+		select(0, NULL, NULL, NULL, &tv);
```

Comment:
I will fix typo here (`tv2` should be the right one) and apply the patch, and BTW we have `-Wall` switch which includes `-Wunused-variable` which should tell you about this error, IOW you never compiled that part of code, right?

Reply:
BTW, maybe we should make this helper to handle EINTR, not sure though

Reply:
> IOW you never compiled that part of code, right?

No, I did, but I had a bunch of commits all in the same branch and I tried to split them up manually... rather than cherry-picking them all (which would have been the smart thing) I tried to manually re-enter this one.